extensions/CoreWidgetProvider/Widgets/SSHWalletWidget.cs

```diff
@@ -3,13 +3,13 @@
 
 using System.Diagnostics;
 using System.Globalization;
-using System.Text.Json;
 using System.Text.Json.Nodes;
-using System.Text.Json.Serialization;
 using System.Text.RegularExpressions;
 using CoreWidgetProvider.Helpers;
 using CoreWidgetProvider.Widgets.Enums;
 using Microsoft.Windows.Widgets.Providers;
+using Newtonsoft.Json;

```

Comment:
There's not a way to do this stuff with System.Text.Json?

Reply:
Good catch! I wasn't sure if I should make use the same JObject types as MergeJsonData(), but we didn't need to because HandleConnect() receives a JSON string from MergeJsonData().